ui/src/server/handlers/api.js

```diff
@@ -3012,6 +3012,33 @@ Fetchr.registerService({
     },
 });
 
+Fetchr.registerService({
+    name: 'resource-access',
+    read(req, resource, params, config, callback) {
+        req.clients.zms.getResourceAccessList(
+            { action: params.action, principal: 'user.' + req.session.shortId },
+            (err, list) => {
+                if (err) {
+                    debug(
+                        `principal: ${req.session.shortId} rid: ${
+                            req.headers.rid
+                        } Error from ZMS while calling getRole API: ${JSON.stringify(
```

Comment:
`getRole` instead of `getResourceAccessList` in error message

Reply:
fixed.